tensorflow/compiler/xla/tests/local_client_aot_test_helper.cc

```diff
@@ -89,7 +89,7 @@ int main(int argc, char** argv) {
   // It's lame to hard-code the buffer assignments, but we need
   // local_client_aot_test.cc to be able to easily invoke the function.
   CHECK_EQ(result->result_buffer_index(), 0);
-  CHECK_EQ(result->buffer_sizes().size(), 3);
+  CHECK_EQ(result->buffer_sizes().size(), 4);
```

Comment:
Chad is right - this should remain "3", otherwise the tests will fail.  If not something really weird is going on.